src/OpenTK.GLWidget/X11/XVisualClass.cs

```diff
@@ -0,0 +1,39 @@
+﻿//
+//  XVisualClass.cs
+//
+//  The Open Toolkit Library License
+//
+//  Copyright (c) 2006 - 2009 the Open Toolkit library, except where noted.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights to
+//  use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+//  the Software, and to permit persons to whom the Software is furnished to do
+//  so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in all
+//  copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+//  EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+//  OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+//  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+//  HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+//  WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+//  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+//  OTHER DEALINGS IN THE SOFTWARE.
+//
+
+namespace OpenTK.X11
+{
+	public enum XVisualClass
```

Comment:
It's part of how X11 handles colours - here's some documentation: https://tronche.com/gui/x/xlib/window/visual-types.html

However, I've just noticed that OpenTK already has all the X-related structures implemented in Platform/X11/API.cs, and they can be used instead. There's one catch - XVisualInfo is set to private. 

I think setting it to internal and dropping these bundled classes and using the existing ones instead is the better option.